### ISSUE TITLE
Meta Pool QF round notion guide

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#1700]
We should create a notion guide like the one we built for [polygon](https://giveth.notion.site/5320f4c94563418e93e4736d9276cba0?v=0d08a51c7da440278cc5971b9001e8b9)

But for MetaPool!

You can see the basic round details & eligibility criteria in this doc: https://docs.google.com/document/d/12GRquZ5Nf99pZnWXzSdiQu8tK0AYxhpvI78VNMNlJMM/edit

As discussed in our call the flow for onboarding will be:
- Open call to ask APAC projects to get on Giveth & apply for verification
- Get projects to fill out a simple typeform [this one, but simplified](https://5ootckqldkl.typeform.com/to/izHxccac) to link to their verified project, and then explain a few more details
- Review projects & add them to the round
- Run the round
- Get projects to add updates as the use the funds (define criteria for update w/ metapool)

So basically, I think we can take the polygon guide... delete a bunch of pages, and edit some.
- edit project eligibilitiy & timeline to summarize content in the google doc above
- Make one page: How to apply for the round - combine create & verify your project....add a bottom section that says... apply via this typeform
- Edit the onboarding AMA section to be like: join our upcoming events, including AMAs & Twitter spaces (because it links to the calendar
- delete "add polygon address"
- keep other pages and make small edits as needed

@oyealmond do you have time to help w/ this tomorrow?

**Comment by oyealmond** [comment]
sure. Working on it.

**Comment by laurenluz** [comment]
praaaaaaaise

**Comment by oyealmond** [comment]
Hey @laurenluz! 
I created How to apply, and the Meta Pool Round information with the Round details. Also erased the categories you told me. [Check the guide here. ](https://www.notion.so/giveth/bd1a4bd498004824b161b81bebf2a995?v=155115c4a54c4285b82c7dc283ed4cc1)


Question, I left How to verify and create a project as 'sub pages' would you prefer to have everything in the "How to apply" page? I think is better in subcategories to make it cleaner. 

Question 2, I saw the donations will be on Optimism, I don't remember but I saw donations will be matched in "all networks" can you clarify in which chain the Round will happen? 

Thanks!

**Comment by laurenluz** [comment]
you're the best A!
